### DISCUSSION TITLE
Recover Github Account

### DISCUSSION THREAD

**Discussion by ghost** [community/community#135738] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

After switching accounts, I deleted my account and then deleted my personal account without realizing it.
My personal account has been deleted. Is it possible to recover it?
If this account were to be lost, I would be very upset because all my efforts over the years would have been lost.
I hope recovery is possible.

**Comment by abdimk** [comment]
"**After switching accounts**,"  you mean your github account ?

**Reply by ghost** [reply]
Yes,  **"After switching accounts,"** mean is github account.

**Comment by abdimk** [comment]
it says unless you have a backup there might be no way to recover it 

https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/deleting-your-personal-account#back-up-your-account-data

**Reply by ghost** [reply]
When I say backup, I guess it's the backup I perform.
Unfortunately, I didn't make a separate backup.
I'm so sad TT
